vcd/datasource_vcd_vapp_vm.go

```diff
@@ -120,6 +120,12 @@ func datasourceVcdVAppVm() *schema.Resource {
 							Computed:    true,
 							Description: "Network card adapter type. (e.g. 'E1000', 'E1000E', 'SRIOVETHERNETCARD', 'VMXNET3', 'PCNet32')",
 						},
+						"connect": {
```

Comment:
I somehow lean towards this name being `is_connected`. That would align with `is_primary` and other places we have `is_fenced`, `is_enabled`, `is_attached`....

Need an agreeement - cc @lvirbalas , @dataclouder

Reply:
I followed `enable` pattern we have. If everyone votes different, I will change.
![image](https://user-images.githubusercontent.com/41415459/87303011-1710fe80-c51b-11ea-806f-ce50bdfa5c93.png)
In UI is `connected`

Reply:
We don't have "enable" pattern, we have `enabled`. So, at least `connected` should be used, or even `is_connected` (needs more thought whether `is_` is needed).

Reply:
Previously we had same discussion - `is_enabled` and we have `enabled`.

Reply:
If it is at least `connected` - it is fine for me. This wording represents more the "desired state configuration"  (end goal) concept:

> After applying I want this NIC to be `connected`.

As opposed to :

> After applying I want to `connect` this NIC

Reply:
renamed